signer/core/cliui.go

```diff
@@ -241,9 +243,28 @@ func (ui *CommandlineUI) OnApprovedTx(tx ethapi.SignTransactionResult) {
 	}
 }
 
+func (ui *CommandlineUI) showAccounts() {
+	accounts, err := ui.api.ListAccounts(context.Background())
+	if err != nil {
+		log.Error("Error listing accounts", "err", err)
+		return
+	}
+	if len(accounts) == 0 {
+		fmt.Printf("No accounts found\n")
+		return
+	}
+	fmt.Printf("\n------- Available accounts -------\n")
+	var out = new(strings.Builder)
+	for i, account := range accounts {
+		fmt.Fprintf(out, "%d. %s at %s\n", i, account.Address, account.URL)
+	}
+	fmt.Printf(out.String())
+}
+
 func (ui *CommandlineUI) OnSignerStartup(info StartupInfo) {
-	fmt.Printf("------- Signer info -------\n")
+	fmt.Printf("\n------- Signer info -------\n")
```

Comment:
```suggestion
	fmt.Print("\n------- Signer info -------\n")
```
think this will fix the linting